storage_manager_new.py

```diff
@@ -5,46 +5,50 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.load_items_from_file()
 
     def load_items_from_file(self):
-        Item.instances = []
+        self.items = []
         File = open(Storage.database_file, 'r')
         for line in File.readlines():
             line = line.rstrip('\n')
-            Item.instance_from_string(line)
+            self.items.append(Item.create_instance_from_string(line))
 
     def add_item(self):
```

Comment:
Feels that this method does much more in it than what it declares in name, maybe we should separate it into two methods: one that actually just adds item, and second that handles all communication with user and just calls this "add item" one?

Reply:
I now realized - not even two separate methods but also the one that handles communication with user, should be also `Iterface` class method. We don't want to handle any user interaction in `Storage` as it is simply not job for this class and we have perfect class for that called `Interface`, right?
So, I think there should be methods in interface that will handle communication and these will call `Storage` specific methods when it comes to adding/deleting etc.